### PR TITLE
changes to episode 7, tasks 1-24

### PR DESCRIPTION
**Pull request by mallewellyn**
Changes all the way down to the end of the section on identifying clusters.

Many minor changes. Major changes to 
- order in which the cluster visualisation sections are presented,
- making sure analysis of figures is presented after figures,
- clarify that motivation for this type of clustering is when we don't know the number of clusters 

tasks 1-24 of #120

### CODE REVIEW

File: _episodes_rmd/07-hierarchical.Rmd [hunk 1]
```diff
@@ -263,30 +271,70 @@ dist_m <- dist(example_data, method = "euclidean")
...
+
+
+# Dendogram visualisation
+
+We can first visualise cluster membership by highlight branches in dendograms. 
+In this example, we calculate a distance matrix between
+samples in the `methyl_mat` dataset. We then draw boxes round clusters obtained with `cutree`.
+
+```{r plot-clust-method}
+## create a distance matrix using euclidean method
```
Comment (alanocallaghan) [inline _episodes_rmd/07-hierarchical.Rmd:303]:
```suggestion
## create a distance matrix using euclidean distance
```

for clarity between `distance` and `method`

**Comment by alanocallaghan** [comment]
Looks good to me after a brief scan but I'll revisit later for a more thorough look

**Review by alanocallaghan — APPROVED** [review]
I merged main into this so please do let me know if I messed any of the conflicts up, but I should've chosen this branch's wherever there was conflict.

Great contribution as always

**Comment by mallewellyn** [comment]
Thanks Alan! Glad I could help - all looks fine to me

**Pull request merged**